oxidation/client/src/views/HomePage.vue

```diff
@@ -50,22 +133,136 @@
 import {
   IonContent,
   IonPage,
+  IonCard,
+  IonCardContent,
+  IonCardTitle,
   IonButton,
   IonIcon,
-  isPlatform,
+  IonRow,
+  IonCol,
+  IonGrid,
   modalController
 } from '@ionic/vue';
 import {
   add,
   link,
-  qrCodeSharp
+  chevronBackOutline,
+  logIn
 } from 'ionicons/icons'; // We're forced to import icons for the moment, see : https://github.com/ionic-team/ionicons/issues/1032
 import { useI18n } from 'vue-i18n';
+import { ref } from 'vue';
 import JoinByLinkModal from '@/components/JoinByLinkModal.vue';
 import CreateOrganization from '@/components/CreateOrganizationModal.vue';
+import OrganizationCard from '@/components/OrganizationCard.vue';
+import PasswordInput from '@/components/PasswordInput.vue';
 import { createAlert } from '@/components/AlertConfirmation';
+import { AvailableDevice } from '../plugins/libparsec/definitions';
 
-const { t } = useI18n();
+export interface DeviceLocalStorageData {
+    slug: string;
+    lastLogin: Date;
+}
+
+const { t, d } = useI18n();
+const deviceList: AvailableDevice[] = [
+  {
+    organizationId: 'MegaShark',
+    humanHandle: 'Maxime Grandcolas',
+    deviceLabel: 'device_label',
+    keyFilePath: 'key_file_path',
+    deviceId: 'device_id',
+    slug: 'slug1',
+    ty: {tag: 'Password'}
+  },
+  {
+    organizationId: 'Resana',
+    humanHandle: 'Maxime Grandcolas',
+    deviceLabel: 'device_label',
+    keyFilePath: 'key_file_path',
+    deviceId: 'device_id',
+    slug: 'slug2',
+    ty: {tag: 'Password'}
+  },
+  {
+    organizationId: 'Oxymore',
+    humanHandle: 'Maxime Grandcolas',
+    deviceLabel: 'device_label',
+    keyFilePath: 'key_file_path',
+    deviceId: 'device_id',
+    slug: 'slug3',
+    ty: {tag: 'Password'}
+  },
+  {
+    organizationId: 'Eddy',
+    humanHandle: 'Maxime Grandcolas',
+    deviceLabel: 'device_label',
+    keyFilePath: 'key_file_path',
+    deviceId: 'device_id',
+    slug: 'slug4',
+    ty: {tag: 'Password'}
+  }
+];
+let selectedDevice: AvailableDevice;
+const password = ref('');
+const showOrganizationList = ref(true);
+
+const deviceLocalStorageDataList = [
+  {slug: 'slug1', lastLogin: new Date('01/11/2023')},
+  {slug: 'slug2', lastLogin: new Date('01/12/2023 12:03:05')},
+  {slug: 'slug3', lastLogin: new Date('01/12/2023 15:12:04')}
+];
+
+function getDeviceLocalStorageData(deviceSlug: string): DeviceLocalStorageData {
+  return deviceLocalStorageDataList.find((device) => {
+    return device.slug === deviceSlug;
+  });
```

Comment:
bad typing, `deviceLocalStorageDataList.find` can be undefined.

We expect that it will never be undefined so add
`  }) as DeviceLocalStorageData;
`